### PR TITLE
Scenario get existing shopcart

### PR DESCRIPTION
**Pull request by carpetri**
All test should pass.

behave -n 'Get an existing Shopcart'

### CODE REVIEW

File: app/static/src/components/Shopcart.vue [hunk 1]
```diff
@@ -17,11 +17,8 @@
             </div>
 
             <div class="col-sm">
-              <input type="number"
-                :id="`product-${product.id}-quantity`"
-                class="form-control form-control-sm"
-                v-model="product.quantity"
-                @keydown.enter="updateProduct(product)" />
+              <h6 class="mb-1" :id="`product-${product.id}-quantity`"
+              v-model="product.quantity">{{ product.quantity}}</h6>
```
Comment (radhikamattoo) [inline app/static/src/components/Shopcart.vue]: For updating a product in a shopcart this has to be an `input` value so you can directly modify the quantity

File: features/steps/shopcart_steps.py [hunk 2]
```diff
@@ -96,14 +96,14 @@ def step_impl(context, message):
...
-    element = context.driver.find_element_by_id('product-' + product_id + '-quantity' )
+    element = context.driver.find_element_by_id('product-' + product_id + '-select' )
     element.clear()
     element.send_keys(int(quantity))
 
 @then(u'I should see "{quantity}" of Product "{product_id}" in the products list')
 def step_impl(context, quantity, product_id):
     assert context.driver.find_element_by_id('shopcart-product-' + product_id)
-    element_value = context.driver.find_element_by_id('product-' + product_id + '-quantity').get_attribute('value')
+    element_value = context.driver.find_element_by_id('product-' + product_id + '-quantity').text
```
Comment (radhikamattoo) [inline features/steps/shopcart_steps.py]: same thing here, it should go back to `.get_attribute('value')`

**Comment by carpetri** [comment]
Works with behave -n 'Get an existing Shopcart' but fails when running all tests
Any suggestions ? @radhikamattoo @jonjrodriguez

**Comment by radhikamattoo** [comment]
@carpetri @jonjrodriguez We're gonna merge everything since they work individually, then we can debug the connection error

**Pull request merged**